### DISCUSSION TITLE
How can I disable R.nvim for .qmd files?

### DISCUSSION THREAD

**Discussion by weberam2** [R-nvim/R.nvim#244] [General]
Title says it all.

I'm using LazyVim. I'm guessing it's going to be something like this?:

```
return {
  "R-nvim/R.nvim",
  opts = {
    ft = ".R, .Rnw, .Rd, .Rmd"
  },
}
```

**Comment by jalvesaq** [comment]
The option for this is controlled by a global variable because `ftplutin/quarto.lua` is sourced by Neovim before `lua/r/config.lua`. Actually, the scripts in the `ftplugin` directory call `require("r.config").real_setup()`.

So, you need this:

```lua
vim.g.R_filetypes = { "r", "rnoweb", "rmd", "rhelp" }
```

This information is buried in the section _6.30. Disable R.nvim commands_.
